openmetadata-docs/content/deployment/docker/volumes.md

```diff
@@ -2,52 +2,115 @@
 title: Docker Volumes
 slug: /deployment/docker/volumes
 ---
+## Advance section
 
-Named volumes can persist data after we restart or remove a container. Also, it’s accessible by other containers.
+Volumes provide the ability to connect specific filesystem paths of the container back to the host machine. If a directory or a file in the container is mounted, changes in that directory  or file can also be seen on the host machine.we are going to use a mapping of a directory present on the host macine with the container path.
 
-For example:
+## Volumes for MYSQL container:
+Following are the changes we have to do while mounting the directory for mysql in OpenMetadata
+- Create a directory to keep your MySQL data or files in the host machine
+```commandline
+mkdir -p /opt/openmetadata/db
+```
+- Update or add the volume in the docker-compose.yml file
+Open the file `docker-compose.yml` downloaded from the Release page [Link](https://github.com/open-metadata/OpenMetadata/releases/download/x.x.x-release/docker-compose.yml) .
 
-```yaml
-version: 3.8
+```commandline
+version: "3.9"
 services:
-  db:
-    image:mysql
-    restart:always
+  mysql:
+    container_name: openmetadata_mysql
+    image: openmetadata/db:0.11.5
+    restart: always
     environment:
-       MYSQL_ROOT_PASSWORD: root
-       MYSQL_DATABASE: test_db
-    ports:
-      - "3306:3306"
+      MYSQL_ROOT_PASSWORD: password
+    expose:
+      - 3306
     volumes:
-      - db_data:/var/lib/mysql
-  volumes:
-    db_data
+     - /opt/openmetadata/db:/var/lib/mysql
+    networks:
+      app_net:
+        ipv4_address: 172.16.240.10
 ```
+## Volumes for PostgreSQL container:
+Following are the changes we have to do while mounting the directory for postgressql in OpenMetadata
+- Create a directory to keep your PostgresSQL data or files in the host machine
+```commandline
+mkdir -p /opt/openmetadata/db
+```
+- Update or add the volume in the docker-compose.yml file
+Open the file `docker-compose.yml` downloaded from the Release page [Link](https://github.com/open-metadata/OpenMetadata/releases/download/x.x.x-release/docker-compose.yml) .
```

Comment:
The link is not a valid one! This will fail when use clicks on it to navigate!

Reply:
Done